environment.yml

```diff
@@ -0,0 +1,11 @@
+name: courses
```

Comment:
Is this the literal name of the repo? Something like `scitools-courses` might be better if this is a free text field, as that's a bit more descriptive and less likely to have a name collision!

Reply:
I wasn't 100% sure what the name was for, but if it is the name of the environment which is created then yes `scitools-courses` makes sense!